app/src/store/store.ts

```diff
@@ -134,14 +138,14 @@ const store: StoreOptions<RootState> = {
     },
 
     /**
-     *  find a post from DataService by given id
+     *  Find a post from DataService by given id
      */
     loadPost(context, id: string): Promise<Post | undefined> {
       return PostService.findById(id).then((post) => post);
     },
 
     /**
-     *  find posts from DataService by setted parameter
+     *  Find posts from DataService by setted parameter
```

Comment:
They not only find the posts, the posts will also loaded from the server. Find sounds more like if they exist